src/otx/core/model/module/base.py

```diff
@@ -133,12 +133,31 @@ def state_dict(self) -> dict[str, Any]:
         state_dict["meta_info"] = self.meta_info
         return state_dict
 
+    def _get_state_dict_from_ckpt(self, ckpt: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Why don't we expose `load_from_prev_otx_ckpt` directly? Can we introduce appropriate notifications as well?

```suggestion
    def _load_from_otx_1_x_ckpt(self, ckpt: dict[str, Any]) -> dict[str, Any]:
         msg = "You are trying to load the model checkpoint created by OTX 1.*. Please switch it to the OTX 2.0 format by saving it again. ...."
         warnings.warn(msg)
         ...
```

Reply:
We need to consider the checkpoints which was saved from OTX1.X and OTX2.0.

In the case of OTX2.0 (lightning), there is no dict_key named `model`. model state_dict was saved at the
* OTX1.5 : ckpt['model']['state_dict']
* OTX2.0 : ckpt['state_dict']

So, I wanted to make the branch to distinguish the ckpt comes from OTX1.x or 2.0. In addition, in the case of OTX1.x, we need to add proper prefix to the ckpt. That's the reason why I implemented like this.

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/2812/commits/c1a146cc4c63d768a4adca7b670fce1d73747a5c